yellowbrick/features/base.py

```diff
@@ -22,8 +22,17 @@
 
 from yellowbrick.base import Visualizer
 from yellowbrick.utils import is_dataframe
+from yellowbrick.style import resolve_colors
+from yellowbrick.exceptions import YellowbrickValueError
+
 from sklearn.base import TransformerMixin
 
+AUTO = "auto"
+SINGLE = "single"
+DISCRETE = "discrete"
+CONTINUOUS = "continuous"
+
```

Comment:
Have you used Python enums before? I'd like to explore their use here. It would allow us to more easily check if the target type string was valid by parsing the enum and group these constants together in a more accessible way for the user. 

https://docs.python.org/3/library/enum.html

Reply:
I have read the doc. I will try implementing it. Or do you wish to explore it together?

Reply:
Go ahead and take a crack at it!